### PR TITLE
WebAPI: Update Method pages to modern structure (part 26)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
### Summary

Fixes empty param sections.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/document/getboxobjectfor/index.md [hunk 1]
```diff
@@ -22,9 +22,14 @@ Returns a `boxObject` (x, y, width, height) for a specified element.
...
 ```
 
-- _boxObject_ is an `nsIBoxObject`.
+### Parameters
+
 - _element_ is a {{domxref("element","DOMElement")}}.
 
+### Return value
+
+An `nsIBoxObject`.
```
Comment (OnkarRuikar) [inline files/en-us/web/api/document/getboxobjectfor/index.md:31]:
```suggestion
A `nsIBoxObject`.
```

File: files/en-us/web/api/mediakeys/createsession/index.md [hunk 2]
```diff
@@ -25,6 +25,15 @@ createSession()
 createSession(mediaKeySessionType)
 ```
 
+### Parameters
+
+- `mediaKeySessionType`
```
Comment (OnkarRuikar) [inline files/en-us/web/api/mediakeys/createsession/index.md:30]:
```suggestion
- `mediaKeySessionType`  {{optional_inline}}
```

File: files/en-us/web/api/paymentrequest/abort/index.md [hunk 3]
```diff
@@ -20,16 +20,16 @@ interface that might be shown.
...
 ```
 
-### Returns
+### Parameters
 
-{{jsxref('undefined')}}.
+None.
 
-### Parameters
+### Returns value
```
Comment (wbamberg) [inline files/en-us/web/api/paymentrequest/abort/index.md:30]:
```suggestion
### Return value
```

**Review by wbamberg — APPROVED** [review]
👍 nice work @OnkarRuikar !

**Pull request merged**